docs/upgrade-to-v5.md

```diff
@@ -50,3 +50,18 @@ TODO(NODE-4771): serializeFunctions bug fix makes function names outside the asc
 We have set our typescript compilation target to `es2020` which aligns with our minimum supported Node.js version 14+. The following is from the typescript release notes on es2020 support, so it's some of the syntax that can be expected to be preserved after compilation:
 
 > This will preserve newer ECMAScript 2020 features like optional chaining, nullish coalescing, export * as ns, and dynamic import(...) syntax. It also means bigint literals now have a stable target below esnext.
+
+### Remove deprecated ObjectId methods
+
+The following deprecated methods have been removed:
+
+- `ObjectId.prototype.generate`
+  - Instead, generate a new ObjectId with the constructor: `new ObjectId()`
+
+- `ObjectId.prototype.getInc`
+- `ObjectId.prototype.get_inc`
+- `ObjectId.get_inc`
+  - The `static getInc()` is private now since it has a side effect of changing the value of the next `ObjectId` generated, using `new ObjectId()` and inspecting the increment section of the bytes can provide insight into the current increment value.
```

Comment:
I think it's worth putting this change on its own line, because it's another breaking-ish change; and if we are making the users do this manually now, we should have sample code pointing out how

Reply:
Sure thing, and just to capture here, the reason I'm leaving this as something to be done manually is inspired by the spec: https://github.com/mongodb/specifications/blob/master/source/objectid.rst#counter it states drivers MUST not have an accessor for this field.